features.html

```diff
@@ -1,14 +1,358 @@
+  </tr>
+  <tr>
+    <td><img src="/assets/features/smart_editor/highlight_basic.png" width="400" height="150"></td>
+    <td><img src="/assets/features/smart_editor/highlight_unsafe.png" width="400" height="150"></td>
+  </tr>
+</table>
+<p></p>
+<p>The highlighting palette is fully customizable for both Darcula and Light color schemes.</p>
+
+
+
+  <h3>Quick docs and Quick definition</h3>
+<table border="1" cellpadding="5">
+  <tr>
+    <td width="50%" >At any code element, including library functions, invoke <strong>Quick Doc</strong> (<code>Ctrl+Q / F1</code>) to view the documentation popup. If there is a piece of code in the docs, it will be formatted and highlighted properly:</td>
+    <td>To view the implementation or declaration of an element without leaving your current context, call <strong>Quick Definition</strong> by pressing <code>Ctrl+Shift+I / ⌥Space</code>:</td>
+  </tr>
+  <tr>
+    <td><img src="/assets/features/smart_editor/quick_doc.png" width="410" height="240"></td>
+    <td><img src="/assets/features/smart_editor/quick_def.png" width="490" height="240"></td>
+  </tr>
+</table>
+<p></p>
+<p>Another handy option for viewing documentation is the <img src="/assets/features/smart_editor/docsrs.svg" width="16"> gutter icon, which you can see next to the extern crate declarations. Click it, and you will jump right to the particular crate’s description on <a href="docs.rs">docs.rs</a>.</p>
+
+<!--div class="feature-group">
+  <div class="feature half">
+<p>At any code element, including library functions, invoke Quick Doc (Ctrl+Q / F1) to view the documentation popup. If there is a piece of code in the docs, it will be formatted and highlighted properly:</p>
+    <p>img 3</p>
+  </div>
+  <div class="feature half">
+<p>To view the implementation or declaration of an element without leaving your current context, call Quick Definition by pressing Ctrl+Shift+I / ⌥Space:</p>
+    <p>img 4</p>
+  </div>
+  <p>Another handy option for viewing documentation is the <img src="/assets/features/smart_editor/docsrs.svg" width="16"> gutter icon, which you can see next to extern crate declarations. Click it, and you will jump right to the particular crate’s description on docs.rs.</p>
+</div-->
+
+
+  <h3>Hints and Type information</h3>
+<p>Hints for function argument names, types, and lambda parameters are displayed in the editor, and IntelliJ Rust adjusts them as you type. If you prefer not to see some of the hints or hide them entirely, you can always tune that in the settings.</p>
+
+
+<table border="1" cellpadding="5">
+  <tr>
+    <td width="50%" >If you press <code>Ctrl</code> and hover the mouse over a hint, it will turn into an interactive inlay:</td>
+    <td>You can also get the type information by calling <strong>Type Info</strong> (<code>Ctrl + Shift + P / ⌃⇧P</code>) for a selected expression:</td>
+  </tr>
+  <tr>
+    <td><img class="gif" data-wait="true" data-scope="gif-scope" width="500"
```

Comment:
Maybe we can create "include" similar with `_includes/feature-gfy.html` to simplify gif insertions?

Reply:
I've already implemented such include html in master.
See at [gif-img.html](https://github.com/intellij-rust/intellij-rust.github.io/blob/master/_includes/gif-img.html). Note, it should be slightly modified to use absolute width of inserting picture